.github/workflows/test.yml

```diff
@@ -29,3 +29,6 @@ jobs:
       - name: Check formatting with black
         run: |
           black pytradfri examples tests --check
+      - name: Run unit tests
+        run: |
+          pytest tests
```

Comment:
We need to install the `pytradfri` local package before we run the tests.

Reply:
Are you sure about this? Don't think we did this when running unit tests in the previous CI/CD setup.

Reply:
The best is to install the package in a virtual environment and run the tests there. `tox` is a tool that does this.

https://docs.pytest.org/en/stable/goodpractices.html